aws/s3/variables.tf

```diff
@@ -100,3 +100,9 @@ variable "force_destroy" {
   type        = bool
   default     = false
 }
+
+variable "acl" {
+  description = "Canned ACL to apply to the bucket. Support private and public-read."
+  type        = string
+  default     = "private"
```

Comment:
Should it be `private-read` or `private` e?

Reply:
should be `private` only a